errors.go

```diff
@@ -140,6 +140,13 @@ type ErrorResponse struct {
 	Code    int      `json:"code" xml:"code,attr"`
 }
 
+// ErrorResponseProduction is a used to display errors as JSON or XML in production
+type ErrorResponseProduction struct {
```

Comment:
I don’t see a need for a second almost identical structure. If trace is unwanted in production either it shouldn’t be set in production or under before rendering